docs/tempo/website/getting-started/example-demo-app.md

```diff
@@ -1,12 +1,12 @@
 ---
-title: Example setups
+title: Example configurations
```

Comment:
In my opinion the current heading is more accurate here because these are complete setups (working examples)